update README to mention that the content has been reintegrated into … (#493)
ros2/rosdistro-testing: README.md

> The content of this repository is no more being used and has been reintegrated into the upstream repository [ros/rosdistro#19908](https://github.com/ros/rosdistro/pull/19908).

Reviewer: I suggest rewording this. In addition to being simpler English. Two separate sentences are easier for automatic translation to handle than a compound sentence.

```suggestion
This repository is no longer in use.
The content of this repository has been reintegrated into the upstream repository [ros/rosdistro#19908](https://github.com/ros/rosdistro/pull/19908).
```
